app/models/programmes/secondary_certificate.rb

```diff
@@ -14,12 +14,6 @@ def mailer
       SecondaryMailer
     end
 
-    def user_meets_completion_requirement?(user)
```

Comment:
We can safely remove this as the responsibility of checking that the user has completed CS accelerator has been moved into a ProgrammeObjective